docs/geedocs/5.3.1/answer/7160008.html

```diff
@@ -73,13 +71,41 @@ <h5>
             <th class="narrow">Description</th>
             <th>Resolution</th>
           </tr>
+          <tr>
```

Comment:
When I (mistakenly) did this, I found a few more: 393, 1236, 1277, 1292, 1300, & 1301.  Are these not valid?

Reply:
Good catch.
393 --> It was fixed in master: 5.3.2
1236, 1239: Ok
1277: OK
1292: OK
1300: Internal clean up
1301: OK

I have updated the release notes adding the issues i tagged above as OK.